source/_integrations/sensor.markdown

```diff
@@ -69,7 +69,7 @@ The type of data a sensor returns impacts how it is displayed in the frontend. T
 - **volume_storage**: Generic stored volume in L, mL, gal, fl. oz., m³, ft³, or CCF
 - **water**: Water consumption in L, gal, m³, ft³, or CCF
 - **weight**: Generic mass in kg, g, mg, µg, oz, lb, or st
-- **wind_speed**: Wind speed in ft/s, km/h, kn, m/s, or mph
+- **wind_speed**: Wind speed in Bft, ft/s, km/h, kn, m/s, or mph
```

Comment:
Is it correct to shorten Beaufort to `Bft`?

Reply:
Yes it is. It's someone's last name. :)

Reply:
I didn't mean it should not be capitalized, I meant to ask if `Bft` is an accepted abbreviation. Based on the discussion in the implementation PR https://github.com/home-assistant/core/pull/105795, I don't think it is and I think we should use the full name.